applications/asset_tracker_v2/src/main.c

```diff
@@ -411,10 +411,10 @@ static void data_get(void)
 	size_t count = 0;
 
 	/* Set a low sample timeout. If GNSS is requested, the sample timeout will be increased to
-	 * accommodate the GNSS timeout. Use 2 seconds to accommodate for
-	 * neighbour cell measurements that usually takes a few seconds.
+	 * accommodate the GNSS timeout. Use 11 seconds to accommodate for
+	 * neighbour cell measurements that can take up to 10.24 seconds.
 	 */
-	app_module_event->timeout = 2;
+	app_module_event->timeout = 11;
```

Comment:
We could have this line inside the if statement that actually adds ncell to the sample request. So that we use 2 seconds as a default and up it to 11 in case ncell is requested.

Reply:
done